alert/alert.go

```diff
@@ -63,31 +66,38 @@ func SendEmail(config Config, body string) {
 	}
 }
 
-func sendAnEmail(emailMsg ses.Message, recipient string, config Config) error {
-	recipients := []*string{&recipient}
-
+func sendAnEmail(emailMsg types.Message, recipient string, cfg Config) error {
 	input := &ses.SendEmailInput{
-		Destination: &ses.Destination{
-			ToAddresses: recipients,
+		Destination: &types.Destination{
+			ToAddresses: []string{recipient},
 		},
 		Message: &emailMsg,
-		Source:  aws.String(config.ReturnToAddr),
+		Source:  aws.String(cfg.ReturnToAddr),
 	}
 
-	cfg := &aws.Config{Region: aws.String(config.AWSRegion)}
-	if config.AWSAccessKeyID != "" && config.AWSSecretAccessKey != "" {
-		cfg.Credentials = credentials.NewStaticCredentials(config.AWSAccessKeyID, config.AWSSecretAccessKey, "")
-	}
-	sess, err := session.NewSession(cfg)
+	svc, err := createSESService(cfg.AWSRegion, cfg.AWSAccessKeyID, cfg.AWSSecretAccessKey)
 	if err != nil {
-		return fmt.Errorf("error creating AWS session: %s", err)
+		return fmt.Errorf("failed to create SES service: %w", err)
 	}
 
-	svc := ses.New(sess)
-	result, err := svc.SendEmail(input)
+	result, err := svc.SendEmail(context.Background(), input)
 	if err != nil {
-		return fmt.Errorf("error sending email, result: %s, error: %s", result, err)
+		return fmt.Errorf("error sending email, error: %w", err)
```

Comment:
If we don't show `result` we don't need to distinguish error from result.

```suggestion
		return fmt.Errorf("error sending email: %w", err)
```

Reply:
Good one